<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightPropertiesTab</class>
 <widget class="QWidget" name="LightPropertiesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>387</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="tabToggle">
     <property name="text">
      <string>Light</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Positional Light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Directional Light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Spot Light</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="enabledCheckBox">
            <property name="text">
             <string>Enabled</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Ambient:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ambientColorRoot"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Diffuse:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="diffuseColorRoot" stretch=""/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Specular:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="specularColorRoot" stretch=""/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Constant Attenuation:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Linear Attenuation</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Exponential Attenuation:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="advancedLightPropertiesRoot">
            <property name="topMargin">
             <number>10</number>
            </property>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
